### ISSUE TITLE
Dragging prevents pointer move events.

### ISSUE THREAD

**Issue by Cybermaxke** [JetBrains/compose-jb#134]
Affects: `0.2.0-build127`

While dragging it's not possible to receive any other mouse move events, I traced the issue back to the `MouseMotionListener` that is being registered. `mouseMoved` will not be called while dragging is active, `mouseDragged` is called in that case.

I currently have the following temporary workaround:
```kotlin
fun main() = Window(title = "Porygen", size = IntSize(600, 1000)) {
  Root()

  // Get the wrapped ComposeLayer
  val window = AppWindowAmbient.current!!.window.contentPane.getComponent(0)
  // Get the original motion listener
  val listener = window.mouseMotionListeners.first()
  window.addMouseMotionListener(object : MouseMotionListener {
    override fun mouseDragged(e: MouseEvent) {
      // Also trigger mouse moved so we don't get locked
      // out of other pointer events while dragging
      listener.mouseMoved(e)
    }
    override fun mouseMoved(e: MouseEvent) {
    }
  })
}
```

**Comment by wooodenleg** [comment]
Thanks for the `MouseMotionListener` tip 👍 

Any progress on this issue?

**Comment by igordmn** [comment]
Should be fixed in 0.4.0-build190.

Now both Swing's `mouseDragged`/`mouseMoved` events are treated as `Modifier.pointerInput` and `Modifier.pointerMove` events (we plan to integrate `pointerMove` onExit/onEnter into `pointerInput` in the future).

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5599">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
